domains/postgres/init.go

```diff
@@ -40,6 +40,27 @@ func Migration() (*migrate.MemoryMigrationSource, error) {
 					`DROP TABLE IF EXISTS domains`,
 				},
 			},
+			{
+				Id: "domain_2",
+				Up: []string{
+					`CREATE TABLE IF NOT EXISTS invitations (
+						invited_by      VARCHAR(36) NOT NULL,
+						user_id         VARCHAR(36) NOT NULL,
```

Comment:
lets have something like `invitee` or `invitee_user_id` instead of `user_id`.
Can we have these everywhere? 
What do you think about this change?